pkg/test/framework/components/echo/kube/instance.go

```diff
@@ -164,18 +164,42 @@ func createVMConfig(ctx resource.Context, c *instance, cfg echo.Config) error {
 	if err != nil {
 		return err
 	}
-	cmd := []string{
-		"x", "workload", "group", "create",
```

Comment:
Unrelated to this PR, moving forward I think we may want to move our docs away from the `workload group` command. The `workload entry` command is valuable for bootstrapping, but this command hides details about the WorkloadGroup that the user would be better off understanding/manually configuring.

Reply:
Agreed, I would be in favor of removal entirely personally